### ISSUE TITLE
integrate a web version of `pest_debugger`

### ISSUE THREAD

**Issue by tomtau** [pest-parser/site#14]
The [pest_debugger](https://docs.rs/pest_debugger/latest/) core functionality has been extracted to a standalone crate that could be used in frontends other than a command line. It will be cool to have something like [regex101](https://regex101.com/) for pest.

**Comment by alexkeizer** [comment]
I'm new to pest, but I'd like to have a look at this and help out!

Could you describe a bit more what the desired end result is? I saw https://pest.rs/#editor, which closely matches what I thought you meant with "regex101 for pest".

**Comment by tomtau** [comment]
@alexkeizer have you tried the command line version of the pest debugger?

**Comment by alexkeizer** [comment]
I just tried it a bit, but most of all I just noticed that regex101 has an actual step-by-step debugger going on (I only knew it for the match highlighting), so now the reference makes a lot more sense.

So, more specifically, you're looking for something like https://regex101.com/debugger for pest, correct?

**Comment by tomtau** [comment]
Yes, perhaps one more potential way could be tracing in a collapsable HTML tree like in this tool:

https://github.com/fasterthanlime/pegviz

**Comment by tomtau** [comment]
this may be perhaps useful: https://tabatkins.github.io/railroad-diagrams/ 
https://github.com/pest-parser/pest/discussions/803
https://github.com/pest-parser/pest/discussions/807
